roles/nexus-ocp/tasks/setup.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Tasks to setup Nexus the first time
+
+# Upload all groovy scripts we will be using
+- include: declare_script.yml
```

Comment:
maybe `include_tasks` here instead of `include` i think it's deprecated.

Reply:
Incorporated into PR